README.md

```diff
@@ -64,7 +64,7 @@ The table below lists the platforms on which the Data SDK has been tested.
 
 The table below lists the dependencies of the Data SDK.
 
-| Library              | Minimum version |
+| Library              | Recommended version |
 | :------------------- | :-------------- |
```

Comment:
Should the whole table be re-aligned now?

Reply:
It renders well. However, I can improve it.